firetweet/src/main/java/org/getlantern/firetweet/fragment/support/AbsStatusesFragment.java

```diff
@@ -379,24 +376,8 @@ public void onStatusActionClick(StatusViewHolder holder, int id, int position) {
                 if (twitter == null) return;
                 if (status.is_favorite) {
                     twitter.destroyFavoriteAsync(status.account_id, status.id);
-                    //spice
-                    SpiceProfilingUtil.profile(activity, status.account_id, status.id + ",Unfavor,"
-                            + status.account_id + "," + status.user_id + "," + status.reply_count
-                            + "," + status.retweet_count + "," + status.favorite_count + "," + status.timestamp);
-                    SpiceProfilingUtil.log(activity, status.id + ",Unfavor,"
-                            + status.account_id + "," + status.user_id + "," + status.reply_count
-                            + "," + status.retweet_count + "," + status.favorite_count + "," + status.timestamp);
-                    //end
                 } else {
                     twitter.createFavoriteAsync(status.account_id, status.id);
-                    //spice
-                    SpiceProfilingUtil.profile(activity, status.account_id, status.id + ",Favor,"
-                            + status.account_id + "," + status.user_id + "," + status.reply_count
-                            + "," + status.retweet_count + "," + status.favorite_count + "," + status.timestamp);
-                    SpiceProfilingUtil.log(activity, status.id + ",Favor,"
-                            + status.account_id + "," + status.user_id + "," + status.reply_count
-                            + "," + status.retweet_count + "," + status.favorite_count + "," + status.timestamp);
-                    //end
```

Comment:
Favoriting a tweet appears immediately with this removed. We don't need any of this stuff anyway.